app/lib/actions/candlepin/environment/set_content.rb

```diff
@@ -1,15 +1,19 @@
 module Actions
   module Candlepin
     module Environment
-      class SetContent < Candlepin::Abstract
+      class SetContent < Actions::Base
+        middleware.use ::Actions::Middleware::RemoteAction
```

Comment:
What does this one do?

Reply:
`RemoteAction` specifically adds in some user information that Candlepin needs, or at least that's how I'm reading it.

I had to add these middleware options because `Candlepin::Abstract` included the candlepin error propagation middleware, which was causing the task to fail even if I caught the 404 in `SetContent`. So, I remove the inheritance from `Candlepin::Abstract`, changed it to `Actions::Base`, and added the 3 other middleware options.